Awful.apk/src/main/java/com/ferg/awfulapp/ThreadDisplayFragment.java

```diff
@@ -471,20 +471,25 @@ public void onDestroy() {
         getLoaderManager().destroyLoader(Constants.POST_LOADER_ID);
     }
 
-    // TODO: fix deprecated warnings
-    private synchronized void refreshSessionCookie(){
-        if(mThreadView != null){
-        	CookieSyncManager.createInstance(getActivity());
```

Comment:
If this is the only thing using CookieSyncManager, please remove the import as well :)

Reply:
Sure, I'll go through and clean up the imports!